concepts/05 Widgets/Sankey/15 Node Labels/05 Customize Labels.md

```diff
@@ -56,6 +56,62 @@ In the following example, the **customizeText** function is used to add incoming
         </dxi-series>
     </dx-sankey>
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template> 
+        <DxSankey ... >
+            <DxLabel :customize-text="customizeText" />
+        </DxSankey>
+    </template>
+
+    <script>
+    import DxSankey, { DxLabel } from 'devextreme-vue/sankey';
+
+    const weightsReducer = (accumulator, currentValue) => {
+        return accumulator + currentValue.weight;
+    }
+
+    export default {
+        components: {
+            DxSankey,
+            DxLabel
+        },
+        methods: {
+            customizeText(node) {
+                return node.title + " (in: " + node.linksIn.reduce(weightsReducer, 0) + ", " +
+                    + "out: " + node.linksOut.reduce(weightsReducer, 0) + ")";
```

Comment:
I examined the snippet but I don't understand why the previous line ends with **+**, and this line also begins with a **+**. Is it ok?

Reply:
The previous line ends with ', '
Or you mean something else?